### PR TITLE
New package: Lenovo.DisplayControlCenter version 3.0.12161.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111896&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/l/Lenovo/DisplayControlCenter/3.0.12161.0/Lenovo.DisplayControlCenter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageIdentifier: Lenovo.DisplayControlCenter
+PackageVersion: 3.0.12161.0
+Installers:
+- InstallerUrl: https://download.lenovo.com/consumer/monitor/lenovodisplaycontrolcenter_3.0.12161.0.zip
+  Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.16299.0
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: msix
```
Comment (CoolPlayLin) [inline manifests/l/Lenovo/DisplayControlCenter/3.0.12161.0/Lenovo.DisplayControlCenter.installer.yaml:13]:
```suggestion
  NestedInstallerType: appx
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
I don't think it's *msix*

**Comment by Trenly** [comment]
The appx package is not signed correctly -
> Installer failed with exit code: 0x800b0109 : A certificate chain processed, but terminated in a root certificate which is not trusted by the trust provider.

**Comment by upintheairsheep** [comment]
Keep open, on vacation

**Comment by ItzLevvie** [comment]
[Policy] Needs-Author-Feedback

**Comment by upintheairsheep** [comment]
Validation screenshot?

**Comment by stephengillie** [comment]
The URL works fine for me, so I'm not seeing why the error occurred.

**Comment by upintheairsheep** [comment]
Keep open

**Pull request closed without merging**